### ISSUE TITLE
matrixStats::{col,row}Quantiles() dropping support for logical input

### ISSUE THREAD

**Issue by HenrikBengtsson** [PeteHaitch/DelayedMatrixStats#60]
I'm dropping support for non-numeric data types to `matrixStats::{col,row}Quantiles()`, cf. https://github.com/HenrikBengtsson/matrixStats/issues/156.

Running revdep checks, it looks like this will trigger an error in your tests.  Does:

https://github.com/PeteHaitch/DelayedMatrixStats/blob/master/tests/testthat/test_manifest.csv#L25

have to be updated?

I can't think of a good/valid reason for calculating quantiles on a logical vector but I happy to keep it if one exists.  My change was mainly triggered by dropping support for quantiles on character vectors.

**Comment by PeteHaitch** [comment]
Thanks for the heads up, Henrik.
I'm happy to follow your lead and drop support for logical vectors.
 I try to keep **DelayedMatrixStats** in-sync with **matrixStats** (although I've not had much time of late).
I'll make the necessary change ASAP.
What's the ETA on submitting to CRAN?

**Comment by HenrikBengtsson** [comment]
Thanks. I was hoping you had a good argument for keeping support for logicals ;)

> What's the ETA on submitting to CRAN?

I've just spent 3-4 days to get revdepcheck working on an HPC system where `R CMD check` don't have internet access. That's a fun challenge when it comes to testing Bioconductor packages :/  I've finished it, and am thinking about releasing the next version of matrixStats quite soon because otherwise there's a risk it'll take quite a while before I can get to it again.
